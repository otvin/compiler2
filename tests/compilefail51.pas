program compilefail51(output);
{test procedures that try to set a return value}

var i:integer;

procedure addone(var a:integer);
begin
	a := a + 1;
	addone := a;
end;

begin
	i := 0;
	addone(i);
	writeln(i);
end.
