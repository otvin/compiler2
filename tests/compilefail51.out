Semicolon expected in file:tests/compilefail51.pas line:9     column:12 
